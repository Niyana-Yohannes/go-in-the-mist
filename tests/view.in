view
place A5
view
place B10
view
term
