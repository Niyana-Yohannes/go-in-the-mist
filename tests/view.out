J10,.................................................
L3,..........................................xxxxxxx
C17,xxxxxxxx......x......x......x......x......x......
